src/zeit/importer/article.py

```diff
@@ -57,6 +60,15 @@ def _transform(xslt, xml, **kwargs):
     return xslt(xml, **kwargs)
 
 
+def map_access(context, text):
```

Comment:
Das Mapping sollten wir denke ich mit in die http://cms-backend.zeit.de:9000/cms/work/data/access.xml schreiben, da stehen auch die C1-Schreibweisen schon drin.